### ISSUE TITLE
Octo Expansion + v.3.1.0 changes

### ISSUE THREAD

**Issue by frozenpandaman** [frozenpandaman/splatnet2statink#51]
New stat.ink field:
```
{"species": "inkling" or "octoling"}
```

New gear:

| Headgear           | ID |
|:-------------------|:---|
| Conductor Cap      |    |
| Golden Toothpick   |    |
| Null Visor Replica |    |
| Octoling Shades    |    |
| Old-Timey Hat      |    |

| Clothing           | ID |
|:-------------------|:---|
| Neo Octoling Armor |    |
| Null Armor Replica |    |
| Old-Timey Clothes  |    |

| Shoes              | ID |
|:-------------------|:---|
| Neo Octoling Boots |    |
| Null Boots Replica |    |
| Old-Timey Shoes    |    |

**Comment by frozenpandaman** [comment]
Pushed in 8ab71f4. Waiting on "dbless" conversation/stuff to merge into master…
